client/src/components/Pages/Overview.vue

```diff
@@ -0,0 +1,132 @@
+            {{ collection.total }} wedstrijden gevonden
+        </h1>
+        <Card>
+            <table
+                class="text-left w-full border-collapse"
+                v-if="matches.length > 0"
+            >
+                <thead>
+                    <tr>
+                        <TableHeader field="ID" />
+                        <TableHeader field="Naam" />
+                        <TableHeader field="Datum/tijd" />
+                        <TableHeader field="Team 1" />
+                        <TableHeader field="Team 2" />
+                        <TableHeader field="Score" />
+                        <TableHeader field="Kruipscore" />
+                        <TableHeader field="Commentaar" />
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr
+                        v-for="match in matches"
+                        v-bind:key="match.id"
+                        class="hover:bg-gray-300 border-b border-gray-300"
+                    >
+                        <TableColumn>
+                            {{ match.id }}
+                        </TableColumn>
+                        <TableColumn>
+                            {{ match.name }}
+                        </TableColumn>
+                        <TableColumn>
+                            {{ match.start | formatDate }}
+                        </TableColumn>
+                        <TableColumn>
+                            {{ match.results[0].team.name }}
+                        </TableColumn>
+                        <TableColumn>
+                            {{ match.results[1].team.name }}
+                        </TableColumn>
+                        <TableColumn>
+                            {{
+                                `${match.results[0].score}-${match.results[1].score}`
+                            }}
+                        </TableColumn>
+                        <TableColumn>
+                            {{
+                                `${match.results[0].crawl_score}-${match.results[1].crawl_score}`
+                            }}
+                        </TableColumn>
+                        <TableColumn>{{ match.results[0].note }}</TableColumn>
+                    </tr>
+                </tbody>
+            </table>
+            <div
+                v-if="matches.length === 0"
+                class="flex items-center bg-blue-300 text-white text-sm font-bold px-4 py-3"
+                role="alert"
+            >
+                <p>Geen statistieken gevonden</p>
+            </div>
+        </Card>
+        <button
+            v-if="next_page_url"
+            v-on:click="loadMore()"
+            class="block text-center bg-blue-500 text-white text-sm font-bold px-4 py-3 rounded shadow-md hover:no-shadow hover:bg-blue-600"
+        >
+            <span v-if="!loading">Meer resultaten laden</span>
+            <img v-else src="/assets/img/loader.svg" width="25" height="25" />
```

Comment:
The blue icon is placed on a blue button, so the contrast is not optimal. Change the color of the icon to white for example.